### DISCUSSION TITLE
Multi language spell check

### DISCUSSION THREAD

**Discussion by SimonBrandner** [wireapp/wire-desktop#5063] [Ideas]
I've been wondering if it was possible to implement multi-language spell checking as it is a feature I would enjoy. 

As I understand this, to enable spell checking for all languages we have to change this:

https://github.com/wireapp/wire-desktop/blob/5ccf09373f63e35da0a3dd4e197e5ecb36b27a46/electron/src/main.ts#L582-L586

into this:

```ts
contents.session.setSpellCheckerLanguages(contents.session.availableSpellCheckerLanguages); 
```

Although this doesn't seem like a really nice solution. Is a cleaner implementation planned? Should I attempt to implement this myself?

**Comment by ffflorian** [comment]
Hi @SimonBrandner,

> Is a cleaner implementation planned?

Currently we don't plan to support multi-language spell checking. I will leave this open as a feature request though.
